Replicator/RevFinder.cc

```diff
@@ -83,7 +83,9 @@ namespace litecore::repl {
 
     // Actually handle a "changes" (or "proposeChanges") message:
     void RevFinder::handleChangesNow(MessageIn *req) {
-        Assert(req->intProperty(kCollectionProperty, kNotCollectionIndex) == collectionIndex());
+        if (req->intProperty(kCollectionProperty, kNotCollectionIndex) == collectionIndex()) {
+            return; // TODO: Assert once refactored to static handler
+        }
```

Comment:
If we support P2P b/w CBL 3.0 and helium, if the message doesn't have index, should it refer (safely) to a default collection somehow?

Reply:
Side comment: The if logic is backwards from what it should be!!